eZ/Publish/API/Repository/Tests/SearchServiceFulltextEmbedTest.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
```

Comment:
New files should enforce:
```suggestion
 */
 declare(strict_types=1);
```